src/main.ts

```diff
@@ -204,10 +206,13 @@ async function getTsServerRepoResult(
     const replayScriptName = path.basename(replayScriptArtifactPath);
     const replayScriptPath = path.join(downloadDir, replayScriptName);
 
+    const rawErrorsScriptName = path.basename(rawErrorsScriptArtifactPath);
+    const rawErrorsScriptPath = path.join(downloadDir, rawErrorsScriptName);
+
     const lsStart = performance.now();
     try {
         console.log(`Testing with ${newTsServerPath} (new)`);
-        const newSpawnResult = await spawnWithTimeoutAsync(repoDir, process.argv[0], [path.join(__dirname, "utils", "exerciseServer.js"), repoDir, replayScriptPath, newTsServerPath, diagnosticOutput.toString(), prng.string(10)], executionTimeout);
+        const newSpawnResult = await spawnWithTimeoutAsync(repoDir, process.argv[0], [path.join(__dirname, "utils", "exerciseServer.js"), repoDir, replayScriptPath, rawErrorsScriptPath, newTsServerPath, diagnosticOutput.toString(), prng.string(10)], executionTimeout);
```

Comment:
This should no longer be necessary (and will probably cause it to exit and print the usage message).